demisto_sdk/commands/content_graph/tests/parsers_and_models_test.py

```diff
@@ -196,6 +196,7 @@ def run(
         expected_preview_only: Optional[bool] = None,
         expected_marketplaces: Optional[List[MarketplaceVersions]] = None,
         expected_content_items: Dict[str, ContentType] = {},
+        expected_deprecated: Optional[bool] = None
```

Comment:
Optional because its default is `False`?

Reply:
optional because its a common function between multiple tests to check that content parsing objects work as expected, in some of the tests we do not want to check them up.

The value of the `deprecated` will be determined by whether the pack name ends with `(Deprecated)`.